README.md

```diff
@@ -1,67 +1,78 @@
-# Aurae
+<!-- THE DOCUMENT -->
 
-[![Main Build (lint, compile, test)](https://github.com/aurae-runtime/aurae/actions/workflows/001-cargo-install-ubuntu-make-build.yml/badge.svg?branch=main)](https://github.com/aurae-runtime/aurae/actions/workflows/001-cargo-install-ubuntu-make-build.yml) [![Documentation](https://github.com/aurae-runtime/aurae/actions/workflows/036-check-website-documentation-aurae-builder-make-check-docs.yml/badge.svg)](https://github.com/aurae-runtime/aurae/actions/workflows/036-check-website-documentation-aurae-builder-make-check-docs.yml) [![Deploy (aurae.io)](https://github.com/aurae-runtime/aurae/actions/workflows/091-deploy-website-documentation-ubuntu-make-docs.yml/badge.svg?branch=main)](https://github.com/aurae-runtime/aurae/actions/workflows/091-deploy-website-documentation-ubuntu-make-docs.yml)
+![Deploy (aurae.io)] ![Documentation] ![Main Build (lint, compile, test)]
 
-Aurae is a free and open source Rust project which houses a memory-safe systems runtime daemon built specifically for enterprise distributed systems called `auraed`.
+<p align="center">
+  <img src="https://raw.githubusercontent.com/aurae-runtime/aurae/main/docs/assets/logo-1024x1024.png" width="450">
+</p>
 
-The `auraed` daemon can be run as a PID 1 on a Linux kernel and manages containers, virtual machines, and spawning short-lived nested virtual instances of itself for an additional layer of isolation.
+# Summary
 
-### Mission
+[Aurae] is a memory-safe runtime daemon, process manager, and PID-1 initialization
+system designed to remotely schedule processes, containers, and virtual machines
+as well as set node configurations like networking storage.
 
-Aurae is on a mission to be the most loved and effective way of managing workloads on a single piece of hardware. Our hope is that by bringing a better set of controls to a node, we can unlock brilliant higher order distributed systems in the future.
```

Comment:
The mission statement is important as the project is still so early. People need a way to understand the motivation and our vision quickly. 

Can we please keep this language:

> Aurae is on a mission to be the most loved and effective way of managing workloads on a node. Our hope is that by bringing a better set of controls to a node, we can unlock brilliant higher order distributed systems in the future.

Reply:
I will implement it exactly as is.